drivers/SmartThings/samsung-audio/src/handlers.lua

```diff
@@ -39,7 +39,7 @@ end
 
 function CapabilityHandlers.handle_play(driver, device, cmd)
   local ip = device:get_field("ip")
-  CapabilityHandlers.handle_on(driver, device, nil) --turn on if device is off
+  -- CapabilityHandlers.handle_on(driver, device, nil) --on/off is not working for samsung-audio (same issue with cloud DTH)
```

Comment:
So the turn on command doesn't work? Can we investigate this further?

Reply:
Because if it really doesn't work at all, we should disable the 'on' command in the device's profile.

Reply:
> Because if it really doesn't work at all (and there's no way to fix it), we should disable the 'on' command in the device's profile.

It's the same behavior for cloud DTH case. Tester had advised the same a few weeks earlier (to disable 'switch' command), however we want to copy the cloud DTH behavior for user.  So, switch option still there. @lelandblue is aware of it.